### PR TITLE
Filing UI not displaying FED alteration

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#11202

*Description of changes:*
The data section is available in the filings json only for a completed filing. Added a check for that

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
> _Issue #:_ /[bcgov/entity#11202](https://github.com/bcgov/entity/issues/11202)
> 
> _Description of changes:_ The data section is available in the filings json only for a completed filing. Added a check for that
> 
> By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

Could you please post a screenshot of before and after? Or broken and fixed? I'm trying to understand what the expected details are.

File: src/components/Dashboard/FilingHistoryList.vue [hunk 1]
```diff
@@ -556,10 +556,12 @@ export default class FilingHistoryList extends Mixins(
           this.isEffectiveDatePast(effectiveDate)
         )
 
-        item.courtOrderNumber = filing.data.order?.fileNumber || ''
-        item.isArrangement = this.isEffectOfOrderPlanOfArrangement(filing.data.order?.effectOfOrder)
-        item.toLegalType = filing.data.alteration?.toLegalType || null
-        item.fromLegalType = filing.data.alteration?.fromLegalType || null
+        if (item.isCompletedAlteration) {
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList.vue:559]: How about a comment saying that this data is only valid for a completed alteration?

Comment (lekshmimallika-aot) [reply src/components/Dashboard/FilingHistoryList.vue:559]: Sure

**Comment by lekshmimallika-aot** [comment]
> > _Issue #:_ /[bcgov/entity#11202](https://github.com/bcgov/entity/issues/11202)
> > _Description of changes:_ The data section is available in the filings json only for a completed filing. Added a check for that
> > By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).
> 
> Could you please post a screenshot of before and after? Or broken and fixed? I'm trying to understand what the expected details are.

broken
fixed

**Comment by severinbeauvais** [comment]
> _Description of changes:_ The data section is available in the filings json only for a completed filing. Added a check for that

Are you saying that attempting to display data that wasn't there caused the FED alteration to not display at all?

**Comment by lekshmimallika-aot** [comment]
> > _Description of changes:_ The data section is available in the filings json only for a completed filing. Added a check for that
> 
> Are you saying that attempting to display data that wasn't there caused the FED alteration to not display at all?

Ya

**Review by severinbeauvais — APPROVED** [review]

**Review by cameron-freshworks — APPROVED** [review]
Short and sweet, gj!

**Pull request merged**